packages/snap-preact-components/src/components/Molecules/LayoutSelector/LayoutSelector.tsx

```diff
@@ -0,0 +1,138 @@
+/** @jsx jsx */
+import { h, Fragment } from 'preact';
+
+import { observer } from 'mobx-react-lite';
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { defined, mergeProps } from '../../../utilities';
+import { ComponentProps, ListOption, StylingCSS, layoutOption } from '../../../types';
+import { Select, SelectProps } from '../Select';
+import { List, ListProps } from '../List';
+import { RadioList, RadioListProps } from '../RadioList';
+
+const CSS = {
+	LayoutSelector: ({}: Partial<LayoutSelectorProps>) => css({}),
+};
+
+export const LayoutSelector = observer((properties: LayoutSelectorProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+
+	const defaultProps: Partial<LayoutSelectorProps> = {
+		label: 'Layout',
+		type: 'dropdown',
+		selected: properties.options && properties.options.length ? properties.options[0] : undefined,
+	};
+
+	const props = mergeProps('layoutSelector', globalTheme, defaultProps, properties);
+
+	const { options, selected, type, onSelect, label, disableStyles, className, style, styleScript } = props;
+
+	const subProps: SelectSubProps = {
+		Select: {
+			// global theme
+			...globalTheme?.components?.select,
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+		RadioSelect: {
+			// global theme
+			...globalTheme?.components?.radioList,
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+		List: {
```

Comment:
Should this add some default props for this component?
Perhaps `multiSelect: false` and/or `horizontal: true` and/or `hideCheckbox: true`?